### ISSUE TITLE
Create pypi package

### ISSUE THREAD

**Issue by rasheed-rd** [ItsIgnacioPortal/XSStrike-Reborn#11]
**Is your feature request related to a problem? Please describe.**
No pypi package exists for XSStrike.  The following are a couple of reasons why having a pypi package is useful:

- Having a src dir allows someone to import the project and use it as another python package.
- Having it on pypi greatly simplifies  the installation of the tool and its dependencies as it allows someone to import it via pip/poetry/any python package manager.

**Describe the solution you'd like**
Convert XSStrike into a package that can be published to pypi.

@ItsIgnacioPortal if you agree that creating a pypi package for XSStike is useful I can do the work and create a PR for it.

**Comment by ItsIgnacioPortal** [comment]
> @ItsIgnacioPortal if you agree that creating a pypi package for XSStike is useful I can do the work and create a PR for it.

Good proposal @rasheed-rd :) Would it use a Github Action? If that's the case, you can make a PR with the yaml file, and I'll handle the account stuff.

**Comment by Chandra158** [comment]
Hi @rasheed-rd, this seems useful. Any update on this ?

**Comment by rasheed-rd** [comment]
> Hi @rasheed-rd, this seems useful. Any update on this ?

Hi, so I had a project that required I get XSStrike packaged and published ASAP. This lead me to [fork](https://github.com/rasheed-rd/XSStrike) the original project, package it, and publish to [PyPi](https://pypi.org/project/xsstrike/). 

If possible I would like to figure out a way to include the new features added to this fork.

**Comment by ItsIgnacioPortal** [comment]
Done! You can install the pypi package by running
```
python3 -m pip install xsstrike_reborn
```

New versions of this tool will automatically be pushed to pypi 😁

**Comment by ItsIgnacioPortal** [comment]
> If possible I would like to figure out a way to include the new features added to this fork.

You could've done the same but instead of forking the original repo, fork this repo :p

Either way, if anyone's interested in making this a package, feel free to make a pull request. I personally don't see much benefit from it; Not that I don't value package managers, it's just that [I already have a solution for this problem](https://github.com/ItsIgnacioPortal/fupdate). It also seems like making this into a package for pypi requires the entire project to be re-structured, so I'm not looking forward to that 😅

**Comment by rasheed-rd** [comment]
> You could've done the same but instead of forking the original repo, fork this repo :p

You're right I guess I could have :). I think the reason I didn't, was I needed to get something into pypi immediately and I had access to the xsstrike project on pypi. I knew it would require a major restructuring effort which may not have been desirable, so I wanted to keep it separate for the time being. 

For me the reason a package is important is that it allows it to be installed easily as a dependency in other python projects.
